src/hardware/mixer.cpp

```diff
@@ -1132,6 +1132,12 @@ float MixerChannel::GetFramesPerBlock() const
 	return static_cast<float>(mixer.blocksize) * stretch_factor;
 }
 
+double MixerChannel::GetMsPerFrame() const
+{
+	return MillisInSecond / sample_rate_hz;
+	// The double return value is used for PIC timing (which uses doubles)
```

Comment:
Minor: can you move the comment to the preceding line please

Reply:
Thanks! Fixed.